metadata/md5-cache/dev-lang/go-bootstrap-1.4.1

```diff
@@ -0,0 +1,10 @@
+DEFINED_PHASES=compile install postinst prepare test
```

Comment:
Prune these two metadata files, both are referring to ebuilds that are not here.